pkg/controller/backingstore/backingstore_controller.go

```diff
@@ -79,6 +85,10 @@ func Add(mgr manager.Manager) error {
 	if err != nil {
 		return err
 	}
+	err = c.Watch(source.Kind(mgr.GetCache(), &corev1.ConfigMap{}), ownerHandler, &filterForNoobaaOwnerPredicate, &logEventsPredicate)
```

Comment:
wouldn't this will run all backingstores in case of change in the config map? can we run somehow only the pvpool type?

Reply:
This will only happen when the configmap noobaa-config will change, which is not a lot.
in that case the reconcile will trigger, but the actual change is only on the reconcile of the PV pool.